src/powershell/FinOpsToolkit.psm1

```diff
@@ -0,0 +1,274 @@
+data LocalizedData
+{
+    # culture="en-US"
+    ConvertFrom-StringData -StringData @'
+    DeployFinOpsHub = Deploying hub from template '{0}' to resource group '{1}'.
+    ExpandingZip = Expanding zip '{0}'.
+    FoundAsset = Found asset '{0}'.
+    FoundLatestRelease = Found latest release '{0}'.
+    NewDirectory = Creating directory '{0}'.
+    NotLoggedIn = Not logged into Azure. Launching 'Login-AzAccount'.
```

Comment:
Microsoft has a rule to use "sign in" instead of "login". Not sure why Az isn't following that, but let's follow the official guidance.

```suggestion
    NotSignedIn = Not signed into Azure. Launching 'Login-AzAccount'.
```

Reply:
done